lib/request-proxy.js

```diff
@@ -70,7 +75,10 @@ proto.proxyReq = function proxyReq(dest, req, res, opts) {
             headers: headers,
             method: method,
             httpVersion: httpVersion,
-            checksum: ringpop.membership.checksum
+            checksum: ringpop.membership.checksum,
+            ringpopKey: key,
+            ringpopSource: ringpop.whoami(),
+            ringpopDest: dest
```

Comment:
we will eventually do tchannel snooping with Matt's tchannel's packet capturer. ill leave it out for now until we need it.